### PR TITLE
Add prometheus emitter for jobs scheduled duration

### PR DESCRIPTION
**Pull request by maxknee**
Signed-off-by: Max Knee <Max_Knee@comcast.com>

[add yourself]: https://github.com/concourse/governance#individual-contributors
[PR requirements]: https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md#pull-request-requirements

## Changes proposed by this PR

closes # 

Adding `jobsSchedulingDuration` for prometheus.
* [x] done
* [ ] todo

## Release Note

[imperative mood]: https://chris.beams.io/posts/git-commit/#imperative

### CODE REVIEW

File: atc/metric/emitter/prometheus.go [hunk 1]
```diff
@@ -213,6 +214,17 @@ func (config *PrometheusConfig) NewEmitter(attributes map[string]string) (metric
 	})
 	prometheus.MustRegister(jobsScheduling)
 
+	jobsSchedulingDuration := prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace:   "concourse",
+		Subsystem:   "jobs",
+		Name:        "schedulingDuration",
+		Help:        "Duration of jobs being scheduled in milliseconds",
+		ConstLabels: attributes,
+		Buckets:     []float64{1, 60, 180, 300, 600, 900, 1200, 1800, 2700, 3600, 7200, 18000, 36000},
```
Comment (xtremerui) [inline atc/metric/emitter/prometheus.go:223]: I think the first bucket could be 30? I don't think it will schedule a job in 1ms in anyway.

**Comment by xtremerui** [comment]
After checking the code I am not sure what are the proper buckets setting for job scheduling duration actually. 

It was configured to `ms` here
https://github.com/concourse/concourse/blob/master/atc/metric/metrics.go#L196

Now considering that it has changed quite a lot in terms of job scheduling, I feel `ms` for this metric is not accurate anymore. I would like to see some more feedbacks of sample value of it after this PR and update those buckets or the duration unit accordingly.

**Comment by maxknee** [comment]
That sounds fine, question around that - what would you consider? like the buckets are too big? Or is there something else you'd want to measure?

**Comment by xtremerui** [comment]
I think now the job scheduling time is at least by seconds so there won't be value of fall into first couple of buckets of job scheduling duration metric.

**Pull request merged**